### DISCUSSION TITLE
So many problems, when can they converge and be stable?

### DISCUSSION THREAD

**Discussion by mythosking** [microsoft/vscode-cpptools#7079] [General]
I'm a C language developer. I'm using vscode, but there are too many problems with the current version. It's not an "easy-to-use" version. Small projects can barely be used.，but for large C / C + + projects, cpptools can't help me read the code very well. These problems once made me want to give up vscode as an excellent tool.

**Comment by sean-mcmanus** [comment]
What problems are you referring to?
